src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectHandler.cs

```diff
@@ -585,6 +578,11 @@ protected virtual Task<bool> HandleSignOutCallbackAsync()
                         tokenEndpointResponse = await RedeemAuthorizationCodeAsync(tokenEndpointRequest);
                     }
 
+                    if (!string.IsNullOrEmpty(tokenEndpointResponse.Error))
```

Comment:
Note: this is a behavior change as the `TokenResponseReceived` event will no longer be called for errored responses.